### DISCUSSION TITLE
Is there any way to bind keys?

### DISCUSSION THREAD

**Discussion by suddenlyToday** [pallets/click#2310] [Q&A] [answered]
Is there any way to bind keys?

If i want to bind a shortcut key in my cli which is powred by Click.

**Comment by ThiefMaster** [comment]
Click is not an interactive terminal UI (look at `prompt-toolkit` if you want this) but a command line argument parser...

**Reply by suddenlyToday** [reply]
Okay, got it. Thank you.

**Comment by davidism** [comment, marked as answer]
No, Click is a CLI, it executes a command and exits, there's nothing long running that would listen for keys. Maybe you're looking for a terminal UI framework, something like https://github.com/Textualize/textual or https://github.com/prompt-toolkit/python-prompt-toolkit

**Comment by suddenlyToday** [comment]
Yes, that's what I want.
Thank you very much!
